src/server/socket.js

```diff
@@ -152,6 +163,27 @@ function invalidateSimHost() {
     hostSockets[SIM_HOST] = null;
 }
 
+function closeConnections() {
+  var hostType,
+      socket;
+  for (hostType in hostSockets) {
+    if (hostSockets.hasOwnProperty(hostType)) {
```

Comment:
Small nit: by using `Object.keys(hostSockets).forEach()`, we don't need to check for `hasOwnProperty()`